assets/js/script.js

```diff
@@ -0,0 +1,113 @@
+var movieFormEl = $("#movieForm");
+var hassanEl = $("#hassan");
+var movieTitleEl = $("#movieTitle");
+
+var mainSectionEl = $("#mainSection");
+
+var API_KEY = "d4b4aa40";
+
+function getMovieByTitle(title) {
+  // http://www.omdbapi.com/?apikey=d4b4aa40&t=blade%20runner&type=movie
+  return fetch(
+    "http://www.omdbapi.com/?apikey=" + API_KEY + "&t=" + title + "&type=movie"
+  ).then(function (res) {
+    return res.json();
+  });
+}
+
+function handleNoMoviesFound(title) {
+  var div = $(
+    '<div class="alert alert-danger alert-dismissible fade show" role="alert"></div>'
+  );
+  var strong = $(
+    "<strong>We didn't find any movies with the title of " + title + "</strong>"
+  );
+  var button = $(
+    '<button type="button" class="btn-close" data-bs-dismiss="alert" aria-label="Close"></button>'
+  );
+
+  div.append(strong);
+  div.append(button);
+
+  mainSectionEl.append(div);
+}
+
+function renderRatingSlide(rating, active) {
+  var carouselSlideDiv = $(
+    '<div class="carousel-item"><div class="container"><div class="card" style="width: 18rem;"><div class="card-body"><h5 class="card-title">' +
+      rating.Source +
+      '</span></h5><p class="card-text">' +
+      rating.Value +
+      "</p></div></div></div></div>"
+  );
+
+  if (active) {
+    carouselSlideDiv.addClass("active");
+  }
+
+  return carouselSlideDiv;
+}
+
+function renderRatingsCarousel(ratings) {
+  var ratingsSlideOne = renderRatingSlide(ratings[0], true);
+  var ratingsSlideTwo = renderRatingSlide(ratings[1], false);
+  var ratingsSlideThree = renderRatingSlide(ratings[2], false);
```

Comment:
Could this be refactored into a loop at a later stage? Non-blocker.

Reply:
Absolutely could - initially started as a loop but refactored for simplicity.